lib/AdobeMarketingCloud/HttpClient.php

```diff
@@ -215,9 +217,16 @@ protected function decodeResponse($response, $options = array())
         switch ($options['format'])
         {
             case 'json':
-                if ((null === $json = json_decode($response['response'], true)) && ($options['debug'] === true)) {
-                    throw new Exception("Response is not in JSON format: \n\n".print_r($response, true));
+                $json = json_decode($response['response'], TRUE);
+                if ($json === null) {
+                    $debug_printout = "";
```

Comment:
One quick change -- could you change this to `debugPrintout`? Otherwise, it looks good. Thanks!

Reply:
done